src/features/vote/ActiveRequests.tsx

```diff
@@ -84,11 +86,18 @@ const ActiveRequests: FC<Props> = ({
     return () => clearInterval(timer);
   }, []);
 
+  const message = currentSigningMessage(Number(roundId));
+  const hashedMessage = ethers.utils.formatBytes32String(message);
```

Comment:
Same question about the hash as above. Also, should we have `currentSigningMessage` return the hashed message if that's the only one we use in the logic?